enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -400,6 +441,11 @@ def get_redemptions_by_policy_uuid(self, enterprise_customer_uuid, lms_user_id,
 
         return redemptions_by_policy_uuid
 
+    @extend_schema(
+        tags=['Subsidy Access Policy Redemption'],
+        summary='Retrieve redemption.',
+        parameters=[serializers.SubsidyAccessPolicyRedemptionRequestSerializer],
```

Comment:
why use the `parameters` kwarg instead of `request`?

Reply:
request seems to “want” to use the serializer you give it to describe the signature of the request body, which we’re typically not doing for many of our GET views.
If we were filtering in the views, we could rely on the view’s filterset_class …we might be able to lightly refactor some existing views to do that instead.
the parameters kwarg of extend_schema lets you pass in OpenApiParameter() objects and optionally a serializer class or instance from which fields will be treated as the available query params.
We can circle back to the finer points of this discussion once we eliminate any of the superfluous views that currently exist for access policy - I expect that the list() and credits_available() methods actually aren’t needed, but I have to re-read the spec and chat with Adam and Marlon about exactly what’s needed.